doc/src/modules.md

```diff
@@ -254,13 +254,16 @@ syntax for formatting and links.
 For modules needing more general documentation, add a `description` to
 `modules/«module»/meta.nix`:
 
-```nix
-description = ''
-  Consider describing which applications are themed by this module (if it's not
-  obvious from the module name), how the applications need to be installed for
-  the module to work correctly, which theming items are supported (colors,
-  fonts, wallpaper, ...), and any caveats the user should be aware of.
-'';
+You can build and view the documentation by running `nix run .#doc`, or
+`serve-docs` from within the dev shell.
+
+```markdown
+# Module Name
+
+Consider describing which applications are themed by this module (if it's not
+obvious from the module name), how the applications need to be installed for the
+module to work correctly, which theming items are supported (colors, fonts,
+wallpaper, ...), and any caveats the user should be aware of.
```

Comment:
Another issue with conflict resolution:

````suggestion
```nix
description = ''
  Consider describing which applications are themed by this module (if it's not
  obvious from the module name), how the applications need to be installed for
  the module to work correctly, which theming items are supported (colors,
  fonts, wallpaper, ...), and any caveats the user should be aware of.
'';
````

Note the "You can build" line is now lower down and reads:

> You can build and view the documentation by running `nix run .#doc`, which will
build the documentation, start a localhost web-server, and open it in your
browser.